tests/unit/lms/services/rsa_key_service_test.py

```diff
@@ -72,10 +108,20 @@ def test_get_random_key(self, svc, valid_keys):
 
     @pytest.fixture
     def valid_keys(self):
-        # Create some noise with some expired keys.
-        factories.RSAKey.create_batch(size=3, expired=True)
         return factories.RSAKey.create_batch(
-            size=3, expired=False, public_key='{"key": "value"}'
+            size=3,
+            expired=False,
+            public_key='{"key": "value"}',
+            created=datetime(2022, 1, 10),
```

Comment:
Some of these could move to the factory, so we don't have to set as many values.

Reply:
I just moved "expired", moving "public_key" seemed too far from the assertion.